winter-website-contact-form.css

```diff
@@ -0,0 +1,122 @@
+.contactMeSection {
+	display: flex;
+	margin: 1rem;
+	text-align: center;
+	display: block;
+}
+
+h1.contactMeText {
+	color: #5d87cf;
+	margin: 3rem 5rem;
+	position: relative;
+}
+
+h2.contactMeTextDetails {
+	color: #d4bdff;
+	margin: 3rem 5rem;
+	font-size: 3rem;
+	position: relative;
+}
+
+form {
+	margin: 10%;
+	padding: 2%;
+	color: #000000;
+	width: 40%;
+	background-color: #7373bf;
+	text-align: left;
+}
+
+input[type=text], input[type=email], input[type=tel] {
+	color: rgb(102,102,186, 0.6);
+	display: block;
+	position: relative;
+	width: 100%;
+	margin: 1% 0 3% 0;
+}
+
+input[type=text], input[type=email], input[type=tel]:after {
+	color: rgb(102,102,186);
+	display: block;
+	position: relative;
+	width: 100%;
+	margin: 1% 0 3% 0;
+}
+
+input[type=submit] {
+	margin-top: 5%;
+	background-color: #cdaaf2;
+	color: #341d42;
+	border-radius: 30%;
+	height: 4rem;
```

Comment:
its generally better to create heights with padding rather than setting a fixed value